src/main/scala/is/hail/HailContext.scala

```diff
@@ -370,16 +379,18 @@ class HailContext private(val sc: SparkContext,
     forceBGZ: Boolean = false,
     headerFile: Option[String] = None,
     nPartitions: Option[Int] = None,
-    dropSamples: Boolean = false): VariantDataset = {
-    importVCFs(List(file), force, forceBGZ, headerFile, nPartitions, dropSamples)
+    dropSamples: Boolean = false,
+    gr: GenomeReference = GenomeReference.defaultReference): VariantDataset = {
```

Comment:
shouldn't we store this in the vds metadata?

Reply:
It's stored in the `vSignature` in the metadata. TVariant is parameterized by the genome reference. However, when importing a VCF, PLINK, BGEN, BED, Interval list etc, we need to know which Genome Reference to parameterize TVariant by.

Reply:
then shouldn't they all have their own references? I just don't see why it needs to be here

Reply:
It seems reasonable to set a default GR for your whole Hail session if you're gonna be working with a few VCFs from the same reference.

That said, I do have a tendency to want users to be explicit about what they want.

Reply:
@danking see 2nd commit. User can set the default reference in Python once HailContext is running.

Reply:
@tpoterba:

 > then shouldn't they all have their own references? I just don't see why it needs to be here

What's "they"?  Each dataset should have its own reference (hence the reference on import_vcf) but we include a default reference if you don't override because the common case (currently at least) is to work with a single reference.